sites/browserpod/src/content/docs/11-reference/00-BrowserPod/00-boot.md

```diff
@@ -0,0 +1,28 @@
+---
+title: boot
+description: Initialize a pod instance
+---
+
+```ts
+class BrowserPod {
+	static async boot(opts: {
+		nodeVersion?: string;
+		apiKey: string;
+	}): Promise<BrowserPod>;
+}
+```
+
+## Parameters
+
+- **opts** (object) - Initialization options passed as an object
```

Comment:
Should we document this method is "expensive"? Or rather mention this consumes tokens?